sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/throughputControl/FluxTests.java

```diff
@@ -0,0 +1,39 @@
+package com.azure.cosmos.implementation.throughputControl;
+
+import org.testng.annotations.Test;
+import reactor.core.publisher.Mono;
+
+public class FluxTests {
```

Comment:
This test doesn't seem to be related to the throughput control

Reply:
oops, definitely a mistake. Removed.